examples.py/Library/PDF Export/LargePage.pyde

```diff
@@ -0,0 +1,23 @@
+"""
+Large Page.
+
+Saves one frame as a PDF with a size larger
+than the screen. When PDF is used as the renderer
+(the third parameter of size) the display window
+does not open. The file is saved to the sketch folder.
+You can open it by "Sketch->Show Sketch Folder."
+"""
+
+add_library('pdf')
+
+
+def setup():
+    size(2000, 2000, PDF, "Line.pdf")
+
+
+def draw():
+    background(255)
+    stroke(0, 20)
+    strokeWeight(20.0)
+    line(200, 0, width / 2, height)
+    exit()
```

Comment:
fixing the wrong commit : https://github.com/jdf/processing.py/pull/249/commits/551e98bd23026f8dafe531b5cfe97e54135093ac